var/spack/repos/builtin/packages/py-us/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyUs(PythonPackage):
+    """US state meta information and other fun stuff."""
+
+    homepage = "https://pypi.org/project/us/"
+    url      = "https://pypi.io/packages/source/u/us/us-1.0.0.tar.gz"
+
+    version('1.0.0', sha256='09dc9ba763e2e4399e6a042104f3e415a7de6bfa4df6f557b4f19e3ba9a22fda')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-jellyfish', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-jellyfish@0.5.6', type=('build', 'run'))
```
You'll also need to add this specific version to `py-jellyfish`.

Reply:
Given this commit: https://github.com/unitedstates/python-us/commit/d57bbee4f8fa7e6b9e8be7e259c62d5273a99778
Are you ok if I put 'py-jellyfish@0.5.6:' (with the colon)? 

If not, I need to add version 0.5.6 to the py-jellyfish package. Or would you rather I put py-jellyfish@0.6.1?

Reply:
When the `setup.py` says `==`, it means if that version isn't already installed, it will automatically install a different version. Since the tarball for 1.0.0 requires 0.5.6, even though newer versions might work, setuptools will automatically download 0.5.6 if you don't already have it installed. You'll have to add 0.5.6 to the jellyfish package.